addons/website/static/src/scss/website.scss

```diff
@@ -1060,6 +1060,26 @@ table.table_desc tr td {
         }
     }
 }
+// Figure with special style
+.figure.o_website_relative_layout {
```

Comment:
Not bad but I would write `.o_figure_relative_layout` ("website" is not needed, you only have to put an app identifier  for specific module (wforum, wevent, ...) and the relative layout makes only sense with figures, so I would have "figure" in the classname. And then you do not have the put `.figure`, which could make the selector too specific for nothing.